### DISCUSSION TITLE
Calling provided guardrails with colang v2

### DISCUSSION THREAD

**Discussion by PeerBoerner1** [NVIDIA-NeMo/Guardrails#602] [General]
After reading the Colang V2 docs, it's not clear to me how to use (call) the provided guardrails like self check input, hallucination detection, and so on. After defining the rails in the config file, regardless of how I use them (in a main flow) or without a flow, I continually get "ValueError: The provided input rail flow `self check input` does not exist".  I'd really appreciate any help the community can provide! Many thanks.

**Comment by drazvan** [comment]
Hi @PeerBoerner1! As per https://docs.nvidia.com/nemo/guardrails/colang_2/overview.html#colang-2-0-beta, the standard library is not yet usable with Colang 2.0. But this will land in the develop branch early next week and will be released in 0.10 in about 3 weeks.

**Reply by PeerBoerner1** [reply]
Thank you!!!
